feature_album/src/main/kotlin/com/igorwojda/showcase/feature/album/domain/model/Album.kt

```diff
@@ -12,5 +12,8 @@ internal data class Album(
     val tracks: List<Track>? = null,
     val tags: List<Tag>? = null,
 ) {
+
+    val id: String = "$artist - $name"
```

Comment:
I defined artificial id, i've checked current data in the app and it looks like that this combination is unique 👌🏻